dist/scripts/app.js

```diff
@@ -1 +1,15 @@
-console.log('hello');
\ No newline at end of file
+'use strict'
```

Comment:
widzę zmiany w dist ale nie widzę w src, nie powinieneś mieć tego w src?

Reply:
To problem z task runnerem zerknę na to. Pisałem o tym do PR ale nie otrzymałem feedback'a